CONTRIBUTING.md

```diff
@@ -14,8 +14,15 @@ low-risk/isolated tasks:
 Developer guidelines
 --------------------
 
-- Nvim developers should read `:help dev`.
+- Nvim developers should read `:help dev` (especially `:help dev-api`).
 - External UI developers should read `:help dev-ui`.
+- API client developers should read `:help dev-api-client`.
+- Nvim developers are _strongly encouraged_ to install `ninja` for faster builds.
+  ```
+  sudo apt-get install ninja-build
+  rm -rf build
```

Comment:
it might be that `.deps` is also needed. #8312

Reply:
Why not suggest `make distclean` here?